src/ODEs/ODETools/DIRungeKutta.jl

```diff
@@ -0,0 +1,140 @@
+"""
+Diagonally Implicit Runge-Kutta ODE solver
+** NOT WORKING YET **
```

Comment:
We cannot put in Gridap work in progress

Reply:
removed in clean version - can implement DIRK on a different branch